src/Npgsql/Util/CodeAnalysis/AllowNullAttribute.cs

```diff
@@ -0,0 +1,11 @@
+﻿#if NET461 || NETSTANDARD2_0 || NETSTANDARD2_1
```

Comment:
Let's combine all these attribute into a single file, NullabilityAttributes.cs, under Npgsql/Util, like it was before...